### PR TITLE
Ability to deploy projects to Vercel with one click

### PR DESCRIPTION
**Pull request by belikers**
Ability to deploy projects to Vercel with one click

### CODE REVIEW

File: src/env/schema.mjs [hunk 1]
```diff
@@ -67,17 +70,12 @@ export const serverEnv = {
...
   GITHUB_CLIENT_SECRET: process.env.GITHUB_CLIENT_SECRET,
-  DISCORD_CLIENT_ID:  process.env.DISCORD_CLIENT_ID,
+  DISCORD_CLIENT_ID: process.env.DISCORD_CLIENT_ID,
   DISCORD_CLIENT_SECRET: process.env.DISCORD_CLIENT_SECRET,
 
   STRIPE_SECRET_KEY: process.env.STRIPE_SECRET_KEY,
   STRIPE_WEBHOOK_SECRET: process.env.STRIPE_WEBHOOK_SECRET,
   STRIPE_SUBSCRIPTION_PRICE_ID: process.env.STRIPE_SUBSCRIPTION_PRICE_ID,
-
-  // Rate limiter
```
Comment (asim-shrestha) [inline src/env/schema.mjs:77]: Hey, looks like this was missed in a merge

**Review by asim-shrestha — CHANGES_REQUESTED** [review]
Looks good besides the other comment, thoughts @awtkns?

File: src/env/schema.mjs (hunk 1 above)
Comment (belikers) [reply src/env/schema.mjs]: Oh, I accidentally missed it

**Comment by TsinghuaSIGS** [comment]
great!

**Pull request merged**